packages/page-storage/src/Selection/Raw.tsx

```diff
@@ -27,30 +36,94 @@ function Raw ({ onAdd }: Props): React.ReactElement<Props> {
 
```

Comment:
in the new naming convention, the argument is rather `keyAsBytes`, no `key` anymore - is that right? it's a bit confusing :stuck_out_tongue:

Reply:
Let me actually go back to previous attempt where `key` was in bytes, I'll add `keyString` instead, as now it is confusing